public/app/features/trails/ActionTabs/BreakdownScene.tsx

```diff
@@ -74,12 +79,64 @@ export class BreakdownScene extends SceneObjectBase<BreakdownSceneState> {
       }
     });
 
+    const trail = getTrailFor(this);
+    trail.state.$timeRange?.subscribeToState(() => {
+      this.clearBreakdownPanelAxisValues();
+    });
+
     const metric = sceneGraph.getAncestor(this, MetricScene).state.metric;
     this._query = getAutoQueriesForMetric(metric).breakdown;
 
     this.updateBody(variable);
   }
 
+  private breakdownPanelMaxValue: number | undefined;
+  private breakdownPanelMinValue: number | undefined;
+  public reportBreakdownPanelData(data: PanelData | undefined) {
+    if (!data) {
+      return;
+    }
+
+    let newMin = this.breakdownPanelMinValue;
+    let newMax = this.breakdownPanelMaxValue;
+
+    data.series.forEach((dataFrame) => {
+      dataFrame.fields.forEach((breakdownData) => {
+        if (breakdownData.type !== FieldType.number) {
+          return;
+        }
```

Comment:
maybe this should also ensure units match? or is this guaranteed?

Reply:
Units are guaranteed to match. Each panel gets the same query with a slightly different label filter.